.github/workflows/dev.yml

```diff
@@ -4,15 +4,18 @@ name: Development build
 
 on:
   workflow_dispatch:
+  pull_request:
```

Comment:
This will fail when the pull request is opened from a fork

Reply:
This is from a fork... Seems to work?

Reply:
Forks does not have access to secrets

Reply:
I see, but using `pull_request_target` should work right? I just can't test it until merged :)

Reply:
```suggestion
  pull_request_target:
````

Reply:
it does, use with care <https://securitylab.github.com/research/github-actions-preventing-pwn-requests/>